src/components/App/index.js

```diff
@@ -8,19 +8,47 @@ import JoinGame from '../JoinGame';
 import Lobby from '../Lobby';
 import Main from '../Main';
 import ErrorScreen from '../ErrorScreen';
+import Cable from 'actioncable';
+
 
 export class App extends Component {
   constructor(){
     super()
     this.state={
-      socket: "0000"
```

Comment:
Question: What is the difference between a `socket` VS `token`

Reply:
The token is string that the user gets back when the get their JWT, and we use the token value as the domain for the websocket